rossum/password.py

```diff
@@ -0,0 +1,25 @@
+import click
+
+from rossum.lib.api_client import RossumClient
+
+
+@click.group("password")
+def cli() -> None:
+    pass
+
+
+@cli.command(name="change", help="Change password.")
+@click.password_option(required=True)  # type: ignore
```

Comment:
It seems to be working well also without required=True

Reply:
if I will remove  -> after entering the command `rossum> password change` it will end with an error:
`TypeError: decorator() missing 1 required positional argument: 'f'`

Reply:
my bad, you are right, I found the reason for this error:
`TypeError: decorator() missing 1 required positional argument: 'f'`